components/form/Select.js

```diff
@@ -5,10 +5,14 @@ export default function Select({
   selected,
 }) {
   return (
-    <select onChange={ev => onChange(ev.target.value)} defaultValue={selected} className={`form-select h-full py-0 pl-2 pr-8 inline-flex justify-center rounded-md border border-gray-300 shadow-sm px-4 py-2 bg-white text-sm font-medium text-gray-700 hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-offset-gray-100 focus:ring-indigo-500 ${className}`}>
+    <select onChange={ev => onChange(ev.target.value)}
+      className={`form-select h-full py-0 pl-2 pr-8 inline-flex justify-center rounded-md border border-gray-300 shadow-sm px-4 py-2 bg-white text-sm font-medium text-gray-700 hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-offset-gray-100 focus:ring-indigo-500 ${className}`}
+      value={selected}
+    >
+      <option value={selected} disabled selected>{selected}</option>
       {
         options.map((option, index) => (
-          <option key={index} value={option.value}>{option.text}</option>
+          <option key={index}>{option.text}</option>
         ))
       }
```

Comment:
@AceTheCreator Something like that (be aware, it's only an example):

```js
      <option key="Filter by tags..." value="">Filter by tags...</option> // other texts
      {
        options.map((option, index) => (
          <option key={index} value={option.text} selected={option.text === selected}>{option.text}</option>
        ))
      }
```

Reply:
@magicmatatjahu are you suggesting we add a clear filter option to the list of options in the select?

Reply:
@AceTheCreator Yes :)

Reply:
@magicmatatjahu  Implemented your suggestion and it was cool :+1:

Reply:
@magicmatatjahu what do you think about the change?

Reply:
I will check that, no worries :)